bitnami/redis/templates/redis-slave-statefulset.yaml

```
@@ -200,6 +200,7 @@ spec:
             {{- end }}
             - name: redis-data
               mountPath: /data
+              subPath: {{ .Values.slave.persistence.subPath }}
```

Since `subPath` is empty by default, we should avoid setting if empty, i.e.:

```suggestion
              {{- if .Values.slave.persistence.subPath }}
              subPath: {{ .Values.slave.persistence.subPath }}
              {{- end }}
```

Could you make that change?

---

Since the default value of subPath in [values.yaml](https://github.com/bitnami/charts/blob/master/bitnami/redis/values.yaml#L744) is `""` and the default value of subPath in [kubernetes](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#volumes-1) is `""` too, maybe there is no need to check again :)

---

I see your point. I was mostly thinking about a user doing `helm template`. However, checking other existing templates, it seems we are doing it this exact same way, so LGTM.